src/Tizen.NUI.Components/Controls/RecyclerView/Layouter/GridLayouter.cs

```diff
@@ -210,11 +210,19 @@ public override void Initialize(RecyclerView view)
 
             bool failed = false;
             //Final Check of FirstIndex
-            while (colView.InternalItemSource.IsHeader(firstIndex) ||
+            if (colView.InternalItemSource.Count - 1 < firstIndex)
```

Comment:
how about changing the condition to be the same with the condition in the while loop?
```
if (colView.InternalItemSource.Count - 1 <= firstIndex)
```

Reply:
This is intentional difference.

if firstIndex is exceed the count is certainly failed,
but if firstIndex is last item, and if that item is not Header,Footer,GroupHeader,GropuFooter, then it can be really our first founded item.
that is why this condition is different.